src/instana/instrumentation/urllib3.py

```diff
@@ -91,7 +95,11 @@ def urlopen_with_instana(
         tracer, parent_span, span_name = get_tracer_tuple()
 
         # If we're not tracing, just return; boto3 has it's own visibility
-        if tracing_is_off() or (span_name == "boto3"):
+        if (
+            tracing_is_off()
+            or (span_name == "boto3")
+            or (agent.options.allow_exit_as_root and "com.instana" in kwargs.get("url"))
```

Comment:
That works, but it will block all spans pointing to any URL containing "com.instana", and this can cause missing spans in the future.

Why not revisit the PR #562 and supress all internal calls?